packages/bsky/tests/views/list-feed.test.ts

```diff
@@ -0,0 +1,194 @@
+import AtpAgent from '@atproto/api'
+import { TestNetwork } from '@atproto/dev-env'
+import { forSnapshot, paginateAll, stripViewerFromPost } from '../_util'
+import { RecordRef, SeedClient } from '../seeds/client'
+import basicSeed from '../seeds/basic'
+import { TAKEDOWN } from '@atproto/api/src/client/types/com/atproto/admin/defs'
+
+describe('pds author feed views', () => {
```

Comment:
Can't wait to get rid of this `pds` prefix some day soon 🤣 
```suggestion
describe('pds list feed views', () => {
```

Reply:
lol yup